fbw-a380x/src/systems/instruments/src/MsfsAvionicsCommon/gauges.tsx

```diff
@@ -390,3 +406,255 @@ export class GaugeComponent extends DisplayComponent<GaugeComponentProps> {
         );
     }
 }
+
+interface GaugeMaxEGTComponentProps {
+    value: Subscribable<number>;
+    x: number;
+    y: number;
+    min: number;
+    max: number;
+    radius: number;
+    startAngle: Subscribable<number>;
+    endAngle: Subscribable<number>;
+    class: string;
+}
+export class GaugeMaxEGTComponent extends DisplayComponent<GaugeMaxEGTComponentProps> {
+    private angle = Subject.create(0);
+
+    private startAngle: number = 0;
+
+    private endAngle: number = 0;
+
+    private value: number = 0;
+
+    private path = Subject.create('');
+
+    private pathTransform = Subject.create('');
+
+    constructor(props: GaugeMaxEGTComponentProps) {
+        super(props);
+
+        this.startAngle = this.props.startAngle.get();
+        this.endAngle = this.props.endAngle.get();
+        this.value = this.props.value.get();
+        this.update();
+    }
+
+    onAfterRender(node: VNode): void {
+        super.onAfterRender(node);
+
+        this.props.startAngle.sub((a) => {
```

Comment:
See other comment, are those actually subjects or values that change during runtime?